doc/source/index.rst

```diff
@@ -328,3 +327,14 @@ jupyter_execute_kwargs
 
     Keyword arguments to pass to ``nbconvert.preprocessors.execute.executenb``, which controls how
     code cells are executed. The default is ``dict(timeout=-1, allow_errors=True)``.
+
+jupyter_sphinx_linenos
+
+    Boolean argument it use line numbering in code cell display by default. Equivalent
```

Comment:
This sentence seems to have a typo.

Reply:
I think saying that it's an argument is redundant. How about the following clarification:

"Whether to show line numbering in all ``jupyter-execute`` sources."